torchci/pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -490,11 +410,18 @@ function GroupedHudTable({
     <GroupFilterableHudTable
       params={params}
       groupNameMapping={groupNameMapping}
-      names={names}
+      names={data.jobNames}
+      groupNames={names}
       expandedGroups={expandedGroups}
       setExpandedGroups={setExpandedGroups}
+      useGrouping={useGrouping}
+      setUserGrouping={setUseGrouping}
```

Comment:
nit: should this be `setUseGrouping`, not `setUserGrouping`?

Reply:
yes! Thanks